logging_demo/package.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format2.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="2">
```

Comment:
Now that https://github.com/ros2/common_interfaces/pull/45 and friends have been merged this should be updated to format3 and be declared as part of the `rosidl_interface_packages` group

Reply:
will do